### PR TITLE
Removed Compile_first option in tests

### PR DESCRIPTION
**Pull request by skandalaCLIMA**
* Removed Compile_first option in tests
* Replaced "spread" function in get_col_dry function in src/rrtmgp/AtmosphericStates.jl

### CODE REVIEW

File: src/rrtmgp/AtmosphericStates.jl [hunk 1]
```diff
@@ -245,32 +244,30 @@ function Base.convert(
...
     mesh_orientation = first(data).mesh_orientation
     gt_0_tropo_lims = first(data).gt_0_tropo_lims
     s_tropo_lims = size(first(data).tropo_lims)
     ngas = length(gas_conc.gas_names)
 
     col_gas = Array{FT}(undef, ncol, nlay, ngas + 1)
-    for i in 1:s[1]
-        for j in 1:s[2]
-            for igas in 1:ngas+1
+    for i = 1:s[1]
```
Comment (charleskawczynski) [inline src/rrtmgp/AtmosphericStates.jl:255]: I think we’ve been mostly using `for i in` and not `for I =` in ClimateMachine.jl. Did you change this? Or was this the formatter?

Comment (skandalaCLIMA) [reply src/rrtmgp/AtmosphericStates.jl:255]: I did not touch this piece of code

Comment (skandalaCLIMA) [reply src/rrtmgp/AtmosphericStates.jl:255]: Looks like the formatter

**Pull request merged**